ewc-rfc004-individual-wallet-attestation.md

```diff
@@ -0,0 +1,176 @@
+# EWC RFC004: Individual Wallet Unit Attestation - v1.0
+
+**Authors:** 
+* Mr Sebastian Bickerle (Lissi GmbH, Germany)
+* Mr Edward Curran (Lissi GmbH, Germany)
+
+**Reviewers:**
+* Dr Andreas Abraham (ValidatedID, Spain)
+* Mr George Padaytti (iGrant.io, Sweden)
+* Mr Lal Chandran (iGrant.io, Sweden)
+* Dr Nikos Triantafyllou (University of the Aegean, Greece)
+
+**Status:**
+
+Current: OAuth 2.0 Attestation-Based Client Authentication Draft 03 [[1](#60reference)]
+
+**Table of Contents**
+- [EWC RFC004: Individual Wallet Unit Attestation - v1.0](#ewc-rfc004-individual-wallet-unit-attestation---v10)
+- [1.0	Summary](#10summary)
+- [2.0	Motivation](#20motivation)
+- [3.0	Wallet Unit Attestation (WUA)](#30wallet-unit-attestation-wua)
+  - [3.1 Wallet Unit Attestation JWT](#31-wallet-unit-attestation-jwt)
+  - [3.2 Wallet Unit Attestation PoP JWT](#32-wallet-unit-attestation-pop-jwt)
+- [4.0	Messages](#40messages)
+  - [4.1 Wallet Unit Attestation within OID4VCI](#41-wallet-unit-attestation-within-oid4vci)
+    - [4.1.1 Token Request](#411-token-request)
+- [5.0	Implementers](#50implementers)
+- [6.0	References](#60references)
+
+# 1.0	Summary
+
+This specification defines a profile for using OAuth 2.0 Attestation-Based Client Authentication to provide Wallet Unit Attestation as defined within the implementing acts for the context of EWC Pilots. 
+
+# 2.0	Motivation
+
+An EUDI Wallet Instance's life cycle begins when the individual installs an EUDI Wallet app provided by an authorized EUDI Wallet Provider. Once installed, the EUDI Wallet Provider issues a Wallet Unit Attestation (WUA), and the instance is now operational [Ref: ARF 1.4, Chapter 6.5.3]. 
+Using the WUA, an electronice attribute attestation (EAA) provider can authenticate a wallet unit and ensure that it is a valid EUDI wallet that has not been revoked before issuing an EAA.
+
+An attestation provider must be able to determine if a wallet instance offers the requireed functional capabilities and meets the security requirements for issuance. For this purpose, the attestation provider must receive an attestation of the wallet’s capabilities and proof of non-revocation for the wallet instance.
+
+# 3.0	Wallet Unit Attestation (WUA)
+
+The attestation of a wallet's capabilities through Wallet Unit Attestation is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03[1].
+
+## 3.1 Wallet Unit Attestation JWT
+
+The following Wallet Unit Attestation JWT must be presented when sending the token request as defined in RFC001[7].
+
+```
+{
+        "typ": "wallet-unit-attestation+jwt",
+        "alg": "ES256",
+        "kid": "1"
+}
+.
+{
+        "iss": "https://attestation-service.com",
+        "sub": "https://wallet-solution.com",
+        "iat": 1541493724,
+        "status": {
+          "status_list": {
```

Comment:
The `status_list` field should not be contained in the JWT being referenced in the status list. The `status` field should have the following structure [1]:
```json
"status": {
  "idx": 0,
  "uri": "https://example.com/statuslists/1"
},
```
[1] https://www.ietf.org/archive/id/draft-looker-oauth-jwt-cwt-status-list-01.html#name-referenced-token-format-and